.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,24 +0,0 @@
----
```

Comment:
Do we need to remove this?

Reply:
No idea, I think GitBook might have run amok here, I'll clean it up.